packages/node_modules/@ciscospark/helper-html/src/html.shim.js

```diff
@@ -109,8 +109,8 @@ function _filterSync(processCallback, allowedTags, allowedStyles, html) {
           node.removeAttribute(attrName);
         }
         else if (attrName === `href` || attrName === `src`) {
-          const attrValue = node.attributes.getNamedItem(attrName).value.trim();
-          if (attrValue.indexOf(`javascript:`) === 0) {
+          const attrValue = node.attributes.getNamedItem(attrName).value.trim().toLocaleLowerCase();
```

Comment:
please use `toLowerCase()` instead of `toLocaleLowerCase()`. We use `toLowerCase()` everywhere else. I'm open to consistently change to `toLocaleLowerCase()`, but IIRC, it actually has unexpected behaviors and doesn't help us in this case (since 'vbscript:' or 'javascript:' aren't localized).